.circleci/config.yml

```diff
@@ -12,10 +12,10 @@ jobs:
             gcc --version
             g++ --version
             apt-get -y update
-            apt-get -y install libclang-3.8-dev clang-3.8 libclang-dev libgcc-6-dev libclang-common-3.8-dev cmake
+            apt-get -y install libclang-dev clang libclang-7-dev libgcc-8-dev libclang-common-7-dev cmake
```

Comment:
Why clang/llvm 7 and not 8?

Reply:
debian buster doesn't have these in the repo. We can add llvm-toolchain-8 manually.